packages/builder/src/components/automation/AutomationBuilder/FlowChart/FlowItemHeader.svelte

```diff
@@ -123,4 +215,47 @@
     display: flex;
     align-items: center;
   }
+
+  .hide-context-actions {
+    display: none;
+  }
+  input {
+    font-family: var(--font-sans);
+    color: var(--ink);
+    background-color: transparent;
+    border: none;
```

Comment:
We should use a 1px solid transparent border here. This will mean that whenever we start typing, and show a blue border, that the input doesn't jump because its size changes all of a sudden. It'll just make the experience a little smoother for the user.